HW01/Ivan_Papushoi/ivan_papushoi_hw_01_t_01.rb

```diff
@@ -0,0 +1,10 @@
+def calc_angle_arrows(hours, minutes)
+  raise 'ValueOutOfRange' if hours > 12 || hours.negative?
+  raise 'ValueOutOfRange' if minutes > 59 || minutes.negative?
```

Comment:
Let's use different error names. We should understand which params are wrong 
```suggestion
  raise 'Minutes value out of range' if minutes > 59 || minutes.negative?
```

Reply:
Done